apps/fm/test/unit/fm_test.js

```diff
@@ -23,14 +23,21 @@ suite('FM', function() {
   suite('frequency dialer', function() {
 
     suiteSetup(function() {
+      favoritesList._save = function() {return true};
+      favoritesList._favList = {};
+
+      historyList._save = function() {return true};
+
       tempNode = document.createElement('div');
       tempNode.id = 'test';
       tempNode.innerHTML =
```

Comment:
If the template here and below are the same, can we take it out as a variable as well and reuse in both setups?